spec/ic.did

```diff
@@ -106,6 +142,15 @@ service ic : {
       cycles: nat;
       idle_cycles_burned_per_day: nat;
   });
+  canister_info : (record {
+      canister_id : canister_id;
+      num_requested_changes : opt nat64;
+  }) -> (record {
+      total_num_changes : nat64;
+      changes : vec canister_change;
```

Comment:
@roman-kashitsyn 's suggestion:
```
changes_total : nat64;
recent_changes : vec canister_change;
```

Reply:
Interface spec meeting says:
- change `changes` to `recent_changes`
- leave `total_num_changes`